src/amo/css/styles.scss

```diff
@@ -1,3 +1,39 @@
 @import '~core/css/inc/mixins';
 @import '~ui/css/vars';
 @import './inc/vars';
+
+*,
+*::before,
+*::after,
+.addon .content .copy,
+.Addon-summary,
+.AddonDescription-contents,
+.AddonSummaryCard-header-text .AddonTitle,
+.AddonTitle,
+.AddonTitle .AddonTitle-author,
+.Card-header,
+.CollectionDetails-title,
+.CollectionDetails-description,
+.Definition-dd,
+.EditableCollectionAddon-name,
+.MetadataCard-list,
+.RatingManager legend,
+.Search .SearchResults-message,
+.SearchContextCard-header,
+.SearchResult-name,
+.ShowMoreCard-contents,
+.UserCollection-name,
+.UserProfile-biography,
+.UserReview,
+.UserReview-body.ShowMoreCard {
```

Comment:
Would it be possible to only use `*`? Also I see that we have a rule in `App/styles.scss`: https://github.com/mozilla/addons-frontend/blob/c7bf0ab9997c8df52b272b9f444340d006ba781a/src/amo/components/App/styles.scss#L30-L35

Obviously that does not work very well, huh. I guess we could still use `overflow-wrap: break-word;` in this rule.